packages/components/psammead-amp-geo/README.md

```diff
@@ -0,0 +1,76 @@
+# psammead-amp-geo - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-amp-geo%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-amp-geo%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/figure--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-amp-geo.svg)](https://www.npmjs.com/package/@bbc/psammead-amp-geo) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `psammead-amp-geo` component is an `<amp-geo>` element to be used for AMP pages. It has a defined set of ISO country groups and we set for our privacy and cookie banners on AMP pages. 
```

Comment:
```suggestion
The `psammead-amp-geo` component is an `<amp-geo>` element to be used for AMP pages. It has a defined set of ISO country groups that we set for our privacy and cookie banners on AMP pages. 
```
Maybe? the sentence didn't really make sense to me otherwise